projects/addon-commerce/pipes/format-card/format-card.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({name: 'tuiFormatCard'})
+export class TuiFormatCardPipe implements PipeTransform {
+    transform(card: string | undefined = '', cardPrefilled: boolean): string {
```

Comment:
`undefined` type is unnecessary with default value, plus default `cardPrefilled` to `false` because people will not need it.

Reply:
fixed